plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Shipping.php

```diff
@@ -45,7 +45,7 @@ public function get_id() {
 	 * @return string
 	 */
 	public function get_title() {
-		return __( 'Get your products shipped', 'woocommerce' );
+		return __( 'Select your shipping options', 'woocommerce' );
```

Comment:
It looks like in the Figma file, the title of the task on LYS sidebar is `Select your shipping options`, while the title of the task on the setup task list is `Get your products shipped`.  🤔

Reply:
Good catch! Let me clarify